probes/activityOnIssuesByCollaboratorsMembersOrOwnersInLast90Days/def.yml

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 OpenSSF Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+id: activityOnIssuesByCollaboratorsMembersOrOwnersInLast90Days
```

Comment:
this is a little bit of a word soup. I understand the want for clarity, but is this too much. thoughts @laurentsimon ?

Reply:
@laurentsimon kind ping.

Reply:
I find the name a bit overloaded now :)
Maybe we can simplify the name and use the `Values` being introduced in the `Finding` structure to indicate 90 days.
Curious if we need to differentiate between collaborators, members and owners in practice. If so, do we need different probes? If not, can we find n umbrella term for all of them, e.g. project maintainers or something like it

Reply:
this one is a tricky one in my opinion. because the 90 day lookback is something inherent in the raw data, not in any policy someone might build on top of it.

Reply:
> Curious if we need to differentiate between collaborators, members and owners in practice. If so, do we need different probes? If not, can we find an umbrella term for all of them, e.g. project maintainers or something like it.

agreed, I think the roles were picked to differentiate between issue activity from someone filing bugs, and responses from the project maintainers.

Reply:
remove 90 days from name, and can always include it later as an informational value if needed like Laurent said.
can combine the collaborator / owner / member bit, as it is more or less an implementation detail.

which leads to a new probe name suggestion: `issueActivityByProjectMember` (or similar)

Reply:
Renamed in https://github.com/ossf/scorecard/pull/3507/commits/d7938f44a1736f9ffb47506ccb021922ef118303